### DISCUSSION TITLE
Launch to play and repeat playlist?

### DISCUSSION THREAD

**Discussion by sm7urw** [hrkfdn/ncspot#1213] [Q&A]
Hi!
Is it possible to launch ncspot from comandline and make it play and repeat a certain playlist?

// HK

**Comment by hrkfdn** [comment]
Yes, via the [IPC socket](https://github.com/hrkfdn/ncspot/blob/main/doc/users.md#remote-control-ipc) you can send a chain of commands, i.e.:
- `repeat list`
- `insert https://open.spotify.com/playlist/$PLAYLIST_ID`
- `play`

**Reply by sm7urw** [reply]
Great!
I red the documentation but I don't understand how to run the command. I've installed via Flatpak. 

Do I run 

_flatpak run io.github.hrkfdn.ncspot - [command] 
_
or how do I send the variables to play the playlist?

Tnx!

// HK
